### PR TITLE
Installation & configuration instructions for tti-lw-cli

### LINKED ISSUE #342: Document installing TTSE CLI
#### Summary

Document installing `tti-lw-cli`, i.e. the TTSE CLI.

#### Why do we need this ?

For customers needing proprietary features via CLI.

#### What is already there? What do you see now?

Only installation of open source, see https://www.thethingsindustries.com/docs/getting-started/cli/installing-cli/

#### What is missing? What do you want to see?

Installation instructions for TTSE CLI.

#### How do you propose to document this?

1. Tabs for Open Source vs Enterprise
2. Installation instructions for Brew (blocked by https://github.com/TheThingsIndustries/lorawan-stack/issues/2712) and Snapcraft (blocked by https://github.com/TheThingsIndustries/lorawan-stack/issues/1700). 

#### Can you do this yourself and submit a Pull Request?

Can review and test

### PR DESCRIPTION
**Pull request by nejraselimovic**
#### Summary
Closes #342 

#### Changes
- [x] installation instructions
- [x] configuration instructions

#### Checklist
- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links. Test with `HUGO_PARAMS_SEARCH_ENABLED=true` if style changes will affect the search bar.
- [ ] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [ ] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

**Comment by nejraselimovic** [comment]
@neoaggelos @johanstokking I provided instructions for installing, but I have troubles with configuring CLI on localhost, so I still need to test that. It's something with self signed certificates again. I'll let you know when this is ready for a review.

**Review by johanstokking — COMMENTED** [review]
Can you test this with `snap` locally? I'm on a Mac where snap doesn't work, and I can't seem to get it working in a Docker container either

File: doc/content/getting-started/cli/installing-cli/_index.md [hunk 1]
```diff
@@ -43,6 +47,47 @@ $ sudo snap refresh ttn-lw-stack
...
+
+```bash
+$ brew upgrade TheThingsIndustries/lorawan-stack/tti-lw-stack
+```
+
+#### Linux
+
+```bash
+$ sudo snap install tti-lw-stack
+$ sudo snap alias tti-lw-stack.tti-lw-cli tti-lw-cli
```
[the remark this answers is not in the document]
Comment (nejraselimovic) [reply doc/content/getting-started/cli/installing-cli/_index.md:72]: this doesn't work (tti-lw-cli snap not found), but tti-lw-cli works once tti-lw-stack is installed with snap

**Comment by johanstokking** [comment]
Indeed that is confusing.

Any ideas @htdvisser?

File: doc/content/getting-started/cli/installing-cli/_index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (johanstokking) [reply doc/content/getting-started/cli/installing-cli/_index.md:72]: Ok!

**Review by johanstokking — APPROVED** [review]

**Comment by htdvisser** [comment]
If I remember correctly, we wanted `tti-lw-cli` to be a drop-in replacement for `ttn-lw-cli`, so that you can get started with open source and then upgrade to enterprise with minimal changes. Same story for `tti-lw-stack` vs `ttn-lw-stack` by the way.

Of course we can update `tti-lw-cli` to look for both `ttn` and `tti` config files...

**Comment by johanstokking** [comment]
> If I remember correctly, we wanted `tti-lw-cli` to be a drop-in replacement for `ttn-lw-cli`, so that you can get started with open source and then upgrade to enterprise with minimal changes. Same story for `tti-lw-stack` vs `ttn-lw-stack` by the way.

Yes, that's indeed the technical explanation. But that may not be entirely clear for people new to this. @nejraselimovic please add this as note to the documentation here.

**Pull request merged**